yolapy/models/user.py

```diff
@@ -0,0 +1,54 @@
+from yoconfig import get_config
+from yolapy.services import Yola
+
+
+class User(object):
+
+    """Yola User."""
+
+    def __init__(self, active=False, deleted=None, email=None, id=None,
```

Comment:
That would make sense if it is a timestamp. It might be. I don't know off the top of my head.